completion_aggregator/signals.py

```diff
@@ -25,22 +25,35 @@ def register():
 
     try:
         from xmodule.modulestore.django import SignalHandler
-        from student.signals.signals import ENROLLMENT_TRACK_UPDATED
-        from openedx.core.djangoapps.course_groups.signals.signals import COHORT_MEMBERSHIP_UPDATED
     except ImportError:
         log.warning(
             "Could not import modulestore signal handlers. Completion Aggregator not hooked up to edx-platform."
         )
     else:
         SignalHandler.course_published.connect(course_published_handler)
         SignalHandler.item_deleted.connect(item_deleted_handler)
-        ENROLLMENT_TRACK_UPDATED.connect(cohort_updated_handler)
+
+    ginkgo_error_template = "%s signal not found. If this is a solutions/ginkgo server, this is expected."
+
+    try:
+        from openedx.core.djangoapps.course_groups.signals.signals import COHORT_MEMBERSHIP_UPDATED
+    except ImportError:
+        log.info(ginkgo_error_template, "COHORT_MEMBERSHIP_UPDATED")
+    else:
         COHORT_MEMBERSHIP_UPDATED.connect(cohort_updated_handler)
```

Comment:
These signals were added after Ginkgo, and I believe represent functionality not used by McKinsey.  @bradenmacdonald can you confirm that McKinsey doesn't need cohorts or enrollment tracks?

Reply:
@jcdyer Yes, as far as I know, they don't use cohorts nor different enrollment tracks.